src/functions/schema.ts

```diff
@@ -60,11 +64,13 @@ export const schema: IFunction<ISchemaOptions> = (targetVal, opts, paths) => {
     const validator = validators.get(schemaObj);
     if (!validator(targetVal) && validator.errors) {
       results.push(
-        ...(betterAjvErrors(schemaObj, targetVal, validator.errors, { format: 'js' }) as IOutputError[]).map(
-          ({ error }) => ({
-            message: cleanAJVErrorMessage(error),
-            path,
-          }),
+        ...(betterAjvErrors(schemaObj, targetVal, validator.errors, { format: 'js' }) as IAJVOutputError[]).map(
+          ({ error, path: errorPath }) => {
```

Comment:
Note to self: add a few tests with `path` assertions. The vast majority of tests covering ajv-based validation are mainly oriented on checking messages, but a valid `path` is crucial as well.